chalice/local.py

```diff
@@ -701,3 +715,11 @@ def __getattr__(self, name):
     def __call__(self, *args, **kwargs):
         # type: (Any, Any) -> Any
         return self._chalice(*args, **kwargs)
+
+
+def base64url_decode(value):
```

Comment:
Why aren't we able to use `base64.urlsafe_b64decode()` directly?  I'd add a comment explaining this.  I'd also move this function into the `LocalGatewayAuthorizer` class.  We can always extract it out later if other parts of the code need it.

Reply:
I basically copied this directly from pyjwt, adjusted to work with strings instead of bytes. It looks weird to me to put this in LocalGatewayAuthorizer, but as you wish. I'll rewrite it to be clearer. I'll have a new version ready later if I have time.